bentoml/_internal/io_descriptors/json.py

```diff
@@ -42,7 +40,7 @@ def default(self, o: _SerializableObj) -> t.Any:
         if isinstance(o, np.ndarray):
             return o.tolist()
 
-        if any(isinstance(o, i) for i in [pd.DataFrame, pd.Series]):
+        if isinstance(o, (pd.DataFrame, pd.Series)):
             raise BentoMLException(
```

Comment:
should we also add support for returning a pydantic model here?

Reply:
I don't think that is the job of a serializer. we should just convert a pydantic model to json

Reply:
yes that's that I meant, if it's a pydantic instance here, convert it to dict

Reply:
it should do it now